### PR TITLE
update dms

### PR DESCRIPTION
**Pull request by qibebt-bioinfo**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by qibebt-bioinfo** [comment]
@aunderwo

**Comment by qibebt-bioinfo** [comment]
@BiocondaBot please add label command

**Comment by qibebt-bioinfo** [comment]
@BiocondaBot I want to upload Dynamic Meta-Storms software package v1.1

**Comment by qibebt-bioinfo** [comment]
@BiocondaBot please add label

**Review by dpryan79 — DISMISSED** [review]

File: recipes/dms/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "dms" %}
+{% set version = "v1.1" %}
+{% set sha256 = "c368b450cdf1a7f95cb835f7b8190305317f53f26cf34e10244ea817bad77da6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/qdu-zyf/dynamic-meta-storms/archive/master.zip
```
Comment (dpryan79) [inline recipes/dms/meta.yaml]: Can you tag a release on either your fork or the original repository?

**Review by dpryan79 — CHANGES_REQUESTED** [review]
We really prefer tagged releases rather than specific commits.

File: recipes/dms/meta.yaml (hunk 1 above)
Comment (qibebt-bioinfo) [reply recipes/dms/meta.yaml]: Yes, I have added the version tag on my fork repository and updated the "meta.yaml" file. Thanks so much!

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by qibebt-bioinfo** [comment]
@dpryan79  Could you please invite me to join the @bioconda orginization on GitHub? Thank you very much.

**Pull request merged**